apis_core/relations/static/js/relation_dialog.js

```diff
@@ -1,3 +1,8 @@
+function tohtml(item) {
+    var $result = $('<span>');
+    $result.html(item.text);
+    return $result;
+}
```

Comment:
```suggestion
function tohtml(item) {
    const span = document.createElement('span');
    span.innerHTML = item.text;
    return span;
}
```
given that we want to get rid of jquery, use plain JS instead of jquery for the formatting